firedrake/mesh.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, mesh, classes, kind, facet_cell, local_facet_number, markers=
 
         self.local_facet_number = local_facet_number
 
+        # assert that markers is a proper subset of unique_markers
+        if markers is not None:
+            for marker in markers:
+                assert(marker in unique_markers)
```

Comment:
Can you add a message to this assertion?  `assert foo, "Message if assertion failed"`